loadimpact/loadimpact.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// An example of usign Pantheon's Quicksilver technology to do 
+// a performance test using Load Impact
+
+// Provide the API Key provided by Load Impact
+// For extra security, you can store this information in
+// the private area of the files directory as documented
+// at https://github.com/pantheon-systems/quicksilver-examples.
+$api_key = 'add-api-key-here';
```

Comment:
I had left the API key defined in the variable since I think it is simpler to get running and the API key *is* stored in a way that isn't web accessible.

Obviously using secrets.json is going to be even better and I added a documentation link to show people how to do this if they want.

Do you (or @greg-1-anderson or @joshkoenig) think we should standardize our examples using secrets.json?